Random Users/ViewControllers/DetailViewController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  DetailViewController.swift
+//  Random Users
+//
+//  Created by Cora Jacobson on 9/12/20.
+//  Copyright © 2020 Erica Sadun. All rights reserved.
+//
+
+import UIKit
+
+class DetailViewController: UIViewController {
+
+    
+    // MARK: - Properties
+    
+    var user: User?
+    
+    private var cache = Cache<String, Data>()
+    private let photoFetchQueue = OperationQueue()
+    private var fetchDictionary: [String: FetchPhotoOperation] = [:]
+    
+    // MARK: - Outlets
+    
+    @IBOutlet weak var imageView: UIImageView!
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var phoneLabel: UILabel!
+    @IBOutlet weak var emailLabel: UILabel!
+    
+    // MARK: - Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        if let user = user {
+            self.updateViews(with: user)
+        }
+    }
+
+    // MARK: - Functions
+    
+    private func updateViews(with user: User) {
+        loadImage()
+        nameLabel.text = "\(user.first) \(user.last)"
+        phoneLabel.text = user.phone
+        emailLabel.text = user.email
+    }
+    
+    private func loadImage() {
```

Comment:
I love that you used an operation here as well. Here's something to think about though. If you had a large app, you would have to make a similar function to this everywhere you wanted to load an image. What way could you think of to make something re-usable so that you wouldn't have to duplicate this code each time?